python/dgl/sampling/randomwalks.py

```diff
@@ -209,6 +210,47 @@ def random_walk(g, nodes, *, metapath=None, length=None, prob=None, restart_prob
     eids = F.from_dgl_nd(eids)
     return (traces, eids, types) if return_eids else (traces, types)
 
+def randomwalk_topk(src, dst, num_samples_per_node, k):
+    """Select the top-k nodes in src for each node in dst(dedup).
```

Comment:
From what I understand, this is fusing `to_simple`, `select_topk`, and counting the number of occurrences together.  The docstring did not show that.  Could you rewrite the docstring?  Also, I don't think `randomwalk_topk` is a suitable name since this function is not related to random walk.

Reply:
Hello, thank you for your reply. I have modified the docstring according to your requirement. I modified these codes to optimize the random walk algorithm at first, so I just give this API with the name "randomwalk_topk". I tried to think of another name but did not get a suitable name, because of my poor English. Could you give me a suggestion about this name?

Reply:
I think this function is only used for PinSAGE.  I'll change it to an internal function instead.